cmd/state-svc/internal/resolver/resolver.go

```diff
@@ -129,3 +133,8 @@ func (r *Resolver) Projects(ctx context.Context) ([]*graph.Project, error) {
 
 	return projects, nil
 }
+
+func (r *Resolver) Quit(ctx context.Context) (<-chan bool, error) {
+	logging.Debug("Quit resolver")
+	return r.done, nil
```

Comment:
As you are returning the `r.done` channel which is the same for every subscriber to this quit event, at most one of the subscribers would receive the `true` value.  But probably none of them (see other comment).  All of them would wake up, which we could argue is enough.  In that case I would change the return channel to `<-chan struct{}` though.

If you actually want to return a `true` event on the channel, I think something like this would work here:
```suggestion
	res := make (chan bool)
	go func() {
		<-r.done
		res <- true
	}()
	return res, nil
```